.github/workflows/python_backend_lint.yml

```
@@ -0,0 +1,78 @@
+name: Lint and Test Python Backend
+
+on: [push, pull_request]
```

```suggestion
on: [push]
```

On push should be sufficient I think, no? If you create a new PR, then the last push will have the workflows run already and this would only waste computing power.

---

Well, for testing it actually makes sense, since the tests then will perform on the files how the would be in the merged state.
But I will split the workflow apart, as you suggested, then only the test part will be performed again

---

Good point!